Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -328,6 +328,7 @@
     - id: WeaponTaser
     - id: WantedListCartridge
     - id: DrinkHosFlask
+    - id: BorgModuleDrozd #_SIS sec borg module
```

В идеале один рецепт - один и тот же коммент. Лучше чтобы весь контент было легко одним сообщением в поиске найти, а не на каждый прототип придумывать ___уникальный___ коммент. Плюс лучше в одном стиле со всем проектом оформлять комменты, а не в разнобой
```suggestion
    - id: BorgModuleDrozd # SIS-Security_Cyborg
```